examples/with-shell-commands/apps/apps-a/README.md

```diff
@@ -0,0 +1,3 @@
+# Top-level package
+
+We'll use this package as the end of your Task Graph. You may want to think of it as the final application you're building (the most common use case for a Turborepo build pipeline).
```

Comment:
"end of your Task Graph"

We should give this a better name. The opposite of "entrypoint". I would also like to the TaskGraph docs here (that we're getting together in the other PR. Perhaps:

```suggestion
We'll use this package as the final node of your [Task Graph](https://turbo.build/repo/docs/core-concepts/task-graph). You may want to think of it as the final application you're building (the most common use case for a Turborepo build pipeline).
```